### DISCUSSION TITLE
Unable to submit scores when "Fade out hit circles earlier" is disabled in Classic

### DISCUSSION THREAD

**Discussion by drkamaon** [ppy/osu#22779] [Q&A] [answered]
That's pretty much the error. Scores are unable to submit when "Fade out hit circles earlier" is disabled.

> ![image](https://user-images.githubusercontent.com/61342323/222724239-f07d3889-ffb2-40e7-b352-91bfeb69fcb3.png)

**Comment by mk56-spn** [comment]
web might not be updated yet to handle it? thats my guess anyways i think its happened before with other mods

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-web/pull/9940
